src/std/env.ab

```diff
@@ -74,3 +75,55 @@ pub fun is_root(): Bool {
 
     return false
 }
+
+pub fun printf(format: Text, args: [Text] = [""]): Null {
+    unsafe ${nameof args}=("{format}" "\$\{{nameof args}[@]}")$
+    unsafe $printf "\$\{{nameof args}[@]}"$
+}
+
+pub fun printf_escape(text: Text): Text {
+    text = unsafe $echo \${nameof text}|sed 's/\\\\/\\\\\\\\/g'$
```

Comment:
wondering if we can do just a single sed call to optimize it

Reply:
maybe with `-e` option